avocado/core/remoter.py

```diff
@@ -46,6 +48,25 @@ class ConnectionError(RemoterError):
     pass
 
 
+def _get_env_vars(env_vars):
+    """
+    Gets environment variables.
+
+    :param variables: A list of variables to get.
+    :return: A dictionary with variables names and values.
+    """
+    LOG.info(env_vars)
+    if env_vars is None:
```

Comment:
You document the `env_vars` variable as a being a list, so, IMO, there's no need for this shortcut here.  More so given that it's an internal utility function.